transformer/edit.go

```diff
@@ -0,0 +1,104 @@
+/*
+ * skogul, edit transformer
+ *
+ * Copyright (c) 2019 Telenor Norge AS
+ * Author(s):
+ *  - Kristian Lyngstøl <kly@kly.no>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301  USA
+ */
+
+package transformer
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"sync"
+
+	"github.com/telenornms/skogul"
+)
+
+// Replace executes a set of regular expressions, in order
```

Comment:
Minor nitpick: I can't see the possibility of supplying multiple regular expressions.

Unless you are thinking of if you add multiple regexes to the same expression. If that's the case, I think it's clear from the supplied regex how it will be executed.

Should the docstring rather be "Replace executes a regular expression" ?

Reply:
You're right, it should. I wrote it before I finished the implementation.